online-store/webpack.config.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
```

Comment:
When I open this file in my code editor it is full of eslint errors. That's not good. Other files are fine